requirements_for_pip.txt

```diff
@@ -21,3 +21,4 @@ six @ file:///opt/concourse/worker/volumes/live/5b31cb27-1e37-4ca5-6e9f-86246eb2
 smmap==3.0.5
 tornado @ file:///opt/concourse/worker/volumes/live/05341796-4198-4ded-4a9a-332fde3cdfd1/volume/tornado_1606942323372/work
 tqdm @ file:///tmp/build/80754af9/tqdm_1611857934208/work
+pyyaml==5.4.1
```

Comment:
@marcinpaluch1994 is this the correct way to add requirements here? Or did you generate these files?

Reply:
I am pretty sure, I created this file with some terminal command:
From readme
"
Alternatively to set up the environment you can use
requirement.txt (created in maxOS with "conda list -e > requirements.txt")
or
requirements_for_pip.txt (created from requirements.txt with "pip freeze > requirements.txt")
)
"

Reply:
Ah wonderful! Thank you for pointing me to that